Passive_Yaw_with_N_Waves/PassiveYawON/wecSimInputFile.m

```diff
@@ -0,0 +1,80 @@
+waves(1).spectrumType = 'PM';
+waves(1).direction = 0;
+waves(1).spread = 1;
+
+marker = 10;
+distance = 10;
+[X,Y] = meshgrid(-marker:distance:marker,-marker:distance:marker);
+waves(1).marker.location = [reshape(X,[],1),reshape(Y,[],1)]; % Marker Locations [X,Y]
+clear('marker','distance','X','Y')
+waves(1).marker.style = 2; % 1: Sphere, 2: Cube, 3: Frame.
+waves(1).marker.size = 20; % Marker Size in Pixels
+waves(1).marker.graphicColor = [1,0,0];
+% 
+ 
+% 
+waves(2) = waveClass('irregular');           % Initialize Wave Class and Specify Type                                 
+waves(2).height = 1;                     % Wave Height [m]
+waves(2).period = 3;                       % Wave Period [s]
+waves(2).spectrumType = 'PM';
+waves(2).direction = 90;
+waves(2).spread = 1;
+% 
+marker = 10;
+distance = 10;
+[X,Y] = meshgrid(-marker:distance:marker,-marker:distance:marker);
+waves(2).marker.location = [reshape(X,[],1),reshape(Y,[],1)]; % Marker Locations [X,Y]
+clear('marker','distance','X','Y')
+waves(2).marker.style = 1; % 1: Sphere, 2: Cube, 3: Frame.
+waves(2).marker.size = 30; % Marker Size in Pixels
+waves(2).marker.graphicColor = [0,0,1];
+
+
+
+%% Body Data
+% Flap NOTE: This test uses unique BEM for the OSWEC
+body(1) = bodyClass('../hydroData/oswec.h5');   % Initialize bodyClass for Flap
+body(1).geometryFile = '../geometry/flap.stl';  % Geometry File
+body(1).mass = 12700;                           % User-Defined mass [kg]
+body(1).inertia = [1.85e6 1.85e6 1.85e6];  % Moment of Inertia [kg-m^2]
+body(1).yaw.option=1;                           % Turn passive yaw ON
+body(1).yaw.threshold=0.01;                     % Set passive yaw threshold
+
+% Base NOTE: This test uses unique BEM for the OSWEC
+body(2) = bodyClass('../hydroData/oswec.h5');   % Initialize bodyClass for Base
+body(2).geometryFile = '../geometry/base.stl';  % Geometry File
+body(2).mass = 999;                             % Placeholder mass for fixed body
+body(2).inertia = [999 999 999];                % Placeholder inertia for fixed body
+body(2).mass = 999;                         % Creates Fixed Body
+body(2).inertia = [999 999 999];  % Moment of Inertia [kg-m^2]
+body(2).yaw.option=1;                           % Turn passive yaw ON
+body(2).yaw.threshold=0.01;                     % Set passive yaw threshold
+
+%% PTO and Constraint Parameters
+% Fixed
+constraint(1)= constraintClass('Constraint1');  % Initialize ConstraintClass for Constraint1
+constraint(1).location = [0 0 -10];                  % Constraint Location [m]
+
+pto(1) = ptoClass('PTO1');                      % Initialize ptoClass for PTO1
+pto(1).stiffness = 0;                                   % PTO Stiffness Coeff [Nm/rad]
+pto(1).damping = 120000;                               % PTO Damping Coeff [Nsm/rad]
+pto(1).location = [0 0 -8.9];                        % PTO Location [m]
+% pto(1).orientation.z=[0 1 0]; % switching so device will yaw
```

Comment:
@salhus Did you want to allow the OSWEC to Yaw based on the multiple wave directions? If not we can probably delete lines 78-79 and relabel this folder as "Multiple_Waves" as we would not be demonstrating the passive yaw feature.

Reply:
@nathanmtom 
Yes, if someone wants to check if the different wave-directions have an impact.

Reply:
@salhus Apologies, last minor comment here but if you want the OSWEC to yaw then lines 79-80 would have to be uncommented. When I run the application with these lines uncommented there is not much yaw motion since the wave directions are either perpendicular or parallel without spreading. 

Recommendation is to either delete these lines and keep the OSWEC rotating about the y-axis as designed or uncomment the lines and have one wave group come from 45 degrees to induce greater yaw motion.

Reply:
Thanks @nathanmtom 
I deleted 78-79 to keep the demo more clear.